modules/about-administrator-perspective.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// web_console/web-console-overview.adoc
+
+:_content-type: CONCEPT
+[id="about-administrator-perspective_{context}"]
+= About the Administrator perspective in the web console
+
+The {product-title} web console provides two perspectives; the *Administrator* perspective and the *Developer* perspective. The *Administrator* perspective enables you to view the cluster inventory, capacity, general and specific utilization information, and the stream of important events, all of which help you to simplify planning and troubleshooting tasks. Both project administrators and cluster administrators can view the *Administrator* perspective.
+
+Cluster administrators can also open an embedded command line terminal instance in the web console in OpenShift Container Platform 4.7 and later. 
```

Comment:
Replace OpenShift Container Platform with {product-title}

Reply:
to open an embedded command line terminal requires `Web Terminal` operator installed 
```
Cluster administrators can also open an embedded command line terminal instance in the web console in OpenShift Container Platform 4.7 and later
```